MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -1104,8 +1108,10 @@
 
 	addMTLPixelFormatCapabilities( iOS_GPUFamily2_v1, RG8Snorm, All );
 
-	addMTLPixelFormatCapabilities( iOS_GPUFamily1_v2, R32Uint, RWC );
-	addMTLPixelFormatCapabilities( iOS_GPUFamily1_v2, R32Sint, RWC );
+	addMTLPixelFormatCapabilities( iOS_GPUFamily1_v2, R32Uint, Write );
+	addMTLPixelFormatCapabilities( iOS_GPUFamily1_v2, R32Sint, Write );
```

Comment:
Was this intentional? You're no longer setting the `Read` and `ColorAtt` caps.

Reply:
It's safe. The `addMTLPixelFormatCapabilities()` function adds (`or`'s) the additional flags to what is there already. The other flags come from the base setting for the format. 

Actually all of the similar `addMTLPixelFormatCapabilities()` calls could do the same, but almost all the other calls are about redefining the capabilities for a Metal feature set table entry it was easier to simply redefine (and visually verify from the Apple docs) the entire entry.

Meh. But having said that...I'll revert the `Write` entry itself to a full redefinition so it's consistent with all the other feature-set-based mods...and then add the `Atomic` part as it's being done here.

Reply:
Oh, you set it above. Never mind.